src/utils/narrow.js

```diff
@@ -155,3 +155,12 @@ export const isSameNarrow = (narrow1: Narrow, narrow2: Narrow): boolean =>
   Array.isArray(narrow1) && Array.isArray(narrow2) && isEqual(narrow1, narrow2);
 
 export const parseNarrowString = (narrowStr: string): Narrow => JSON.parse(unescape(narrowStr));
+
+const starredNarrow: Narrow = [
```

Comment:
You can export this too. We do export both the stringified and the array narrows.